pkg/cliplugins/workload/plugin/sync.go

```diff
@@ -416,13 +460,13 @@ func (o *SyncOptions) getResourcesForPermission(ctx context.Context, config *res
 			return false, nil //nolint:nilerr
 		}
 
-		// skip if there is only the local kubernetes APIExport in the synctarget workspace, since we may not get syncedResources yet.
-		clusterName := logicalcluster.From(syncTarget)
+		if len(syncTarget.Spec.SupportedAPIExports) == 0 {
+			return true, nil
+		}
 
+		// skip if there is only the local kubernetes APIExport in the synctarget workspace, since we may not get syncedResources yet.
```

Comment:
```suggestion
		// skip if there is only the local 'imported-apis' APIExport in the synctarget workspace, since we may not get syncedResources yet.
```